stack.yaml

```diff
@@ -14,46 +14,40 @@ extra-deps:
     https://gitlab.com/morley-framework/morley.git
     # ^ CI cannot use ssh, so we use http clone here
   commit:
-    7e81317ef21f86d17f04b63c1c9679b710fdb54b # master
-    #nix-sha256: 0yy1r9w0jlj11fpfk02p6pn53fcvc4djqflsbd8srz5ngndc57pi
+    9c4583f67fc00738b6f2b6701ce56b4c79f3a55f  # master
   subdirs:
-    - code/indigo
-    - code/lorentz
-    - code/morley
-    - code/morley-ledgers
-    - code/morley-ledgers-test
-    - code/morley-upgradeable
     - code/morley-multisig
     - code/morley-client
-    - code/morley-nettest
+    - code/cleveland
 
 - git:
-    https://gitlab.com/obsidian.systems/tezos-bake-monitor-lib.git
+    https://gitlab.com/morley-framework/morley-ledgers.git
+    # ^ CI cannot use ssh, so we use http clone here
   commit:
-    19a9ce57a0510bc3ad8a3f639d0a968a65024b86
-  # nix-sha256: 0ypf7z2c9w59rz7hymzdyx87783qdfp3kygs9jl8qnmbfslmi8jr
+    7f0cbca64ed57b072ee9befc4332cf6fcfa29e71 # master
   subdirs:
-    - tezos-bake-monitor-lib
+    - code/morley-ledgers
+    - code/morley-ledgers-test
+
+- git:
+    https://gitlab.com/morley-framework/morley-upgradeable.git
+    # ^ CI cannot use ssh, so we use http clone here
+  commit:
+    4af615e1a98cbd315aaab7ce100ad919812dcef3 # master
 
 # Stable parts of morley available from Hackage
 - morley-prelude-0.3.0
 - tasty-hunit-compat-0.2
+- morley-1.7.1
+- lorentz-0.6.1
+- indigo-0.3.0
```

Comment:
I remember you needed newer versions of some packages. Is it needed for another part of this issue?

Reply:
After all, I just ignored the issue I had and did some temporary weird thing: https://github.com/tz-wrapped/tezos-btc/pull/133/files#diff-6132ec64701d409c2b7e6ec432a137eca620b39dd5eb6a19fc405aa25f7457e3R140

With the recent morley version, it should resolve on itself.

Reply:
This link doesn't work for me 😕 

I mean the part after `#` seems to be ignored.

Reply:
:fuuu:

Please grep for `looking into ParameterType`

Reply:
Ah, ok, I see it now. Changes in that file are hidden by default, that's why the link didn't work.